src/libraries/System.IO.FileSystem/tests/ManualTests/ManualTests.cs

```diff
@@ -0,0 +1,90 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using Microsoft.DotNet.XUnitExtensions;
+using Xunit;
+
+namespace System.IO.ManualTests
+{
+    public class FileSystemManualTests
+    {
+        public static bool ManualTestsEnabled => !string.IsNullOrEmpty(Environment.GetEnvironmentVariable("MANUAL_TESTS"));
+
+        [Fact]//[ConditionalFact(nameof(ManualTestsEnabled))]
+        [PlatformSpecific(TestPlatforms.AnyUnix)]
+        public static void Throw_FileStreamDispose_WhenRemoteMountRunsOutOfSpace()
+        {
+            /*
+            Pre-requisites to run this test:
+
+            - The remote machine has a folder ~/share, where a small ext4 1MB drive is mounted.
+            - The remote drive is almost full.
+            - The remote drive has one file, called "copyme.txt", large enough that if we attempt to programatically copy it into
```

Comment:
Are these pre-requisites still valid now that filling of the drive is automated?

Reply:
Not needed, removed.